cli/start.go

```diff
@@ -100,7 +104,15 @@ func MakeStartCommand() *cobra.Command {
 				if err != nil && !errors.Is(err, keyring.ErrNotFound) {
 					return err
 				}
+
 				opts = append(opts, node.WithEncryptionKey(encryptionKey))
+				// WARNING: This relies on the fact that the keyring password must have been entered at least once already
+				opts = append(opts, node.WithKeyring(immutable.Some(kr)))
+			}
+
+			acpType := getAcpType(cfg.GetString("acp.type"))
+			if acpType.HasValue() {
```

Comment:
It says outdated, because I was reviewing commit by commit. But the question still stands. It seems like the default is still local acp correct? The user can't set acp to off yet?

Reply:
> What happens if no acp is configured or explicitly set?

It will use the `node` package default.  `"local"`

Reply:
> > What happens if no acp is configured or explicitly set?
> 
> It will use the `node` package default. `"local"`

All G then, just wanted to confirm